js/src/components/lineage/LineageViewContext.tsx

```diff
@@ -34,8 +33,9 @@ export interface LineageViewContextType {
   viewOptions: LineageDiffViewOptions;
   onViewOptionsChanged: (options: NewType) => void;
 
-  // select
-  selectNodeMulti: (nodeId: string) => void;
+  // Multi nodes selection
+  selectMode?: "selecting" | "action_result";
```

Comment:
would it be better to use `selectMode: "selecting" | "action_result" | undefined;`?
because `selectMode` should not be optional here.

Reply:
Fixed